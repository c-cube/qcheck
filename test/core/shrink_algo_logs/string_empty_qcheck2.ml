open QCheck2

let strings_are_empty =
  Test.make ~name:"strings are empty"
    Gen.string (Log.shrinks_string @@ fun s -> s = "")
;;
try Test.check_exn strings_are_empty with Test.Test_fail _ -> ()
