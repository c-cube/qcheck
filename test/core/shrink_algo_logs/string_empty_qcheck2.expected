fails "(Z\139\244"
holds ""
fails "\146\138"
holds ""
fails "\145"
holds ""
fails "a"
